pkg/dbc/parser.go

```diff
@@ -315,13 +316,19 @@ func (p *Parser) int() int64 {
 		isNegative = true
 	}
 	tok := p.nextToken()
-	if tok.typ != scanner.Int {
-		p.failf(tok.pos, "expected int")
+	if tok.typ != scanner.Int && tok.typ != scanner.Float {
+		p.failf(tok.pos, "expected int or float")
 	}
-	i, err := strconv.ParseInt(tok.txt, 10, 64)
+	f, err := strconv.ParseFloat(tok.txt, 64)
 	if err != nil {
 		p.failf(tok.pos, "invalid int")
 	}
+	i := int64(f)
```

Comment:
should we have some type of check that it's a round number?

Reply:
My logic here is if the attribute keyword is `INT`, which in turns invokes `int()` then we'll treat it like an int. 😁 "If it quacks like a 🦆" etc

Reply:
Good point!